packages/suins/Move.lock

```diff
@@ -25,3 +25,17 @@ dependencies = [
 compiler-version = "1.24.0"
 edition = "2024.beta"
 flavor = "sui"
+
+[env]
+
+[env.mainnet]
```

Comment:
Let's make sure we do proper version tracking here. We're on version 3 iirc. you can use GQL to check it.